packages/vue/src/components/molecules/SfTile/SfTile.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="sf-tile" :style="style">
+    <!-- @slot Use this slot to replace title -->
+    <slot name="title" v-bind="{ title }">
+      <h3 v-if="title" class="sf-tile__title">
+        {{ title }}
+      </h3>
+    </slot>
+  </div>
+</template>
+<script>
+export default {
+  name: "SfAlert",
+  props: {
+    /**
+     * Tile title
+     */
+    title: {
+      type: String,
+      default: "",
+    },
+    /**
+     * Tile background color
+     */
+    background: {
```

Comment:
Imho that should be CSS. You're probably refering to Banner component but in that case setting background is as easy as setting CSS property

```css
.tile-class {
  background: pink;
```